### DISCUSSION TITLE
VS live preview fails with Todo tutorial sample

### DISCUSSION THREAD

**Discussion by sloth77** [AvaloniaUI/Avalonia#11276] [General]
At the following point in the "Todo" tutorial:

https://docs.avaloniaui.net/tutorials/todo-list-app/wiring-up-the-views

When TodoListView.axaml is modified to use "Content="{Binding List}"", the live preview fails to show the checkboxes, it only shows the "Add an item" button at the bottom.  The code compiles and runs fine, i.e. shows the todo list correctly.

![image](https://user-images.githubusercontent.com/2922319/236638515-aee9f77d-7b18-4ace-b948-7a1e2255a662.png)

Thanks

**Comment by timunie** [comment]
You need design time data for the previewer to display correctly.

**Comment by timunie** [comment]
Let me convert this into a discussion at this stage. If we later found its a bug in the designer, we can still transfer the issue.
